cellmlmanip/model.py

```diff
@@ -555,21 +558,29 @@ def get_symbol_by_ontology_term(self, namespace_uri, local_name):
             raise ValueError('Multiple variables annotated with {%s}%s' %
                              (namespace_uri, local_name))
 
-    def _get_symbols_by_rdf(self, predicate, object_=None):
+    def get_rdf_annotations(self, subject=None, predicate=None, object=None):
+        """Searches the RDF graph and returns the triples for the given (subject, predicate, object)"""
```

Comment:
Can we expand this docstring to be more informative? So the one-liner could end 'triples matching the given parameters', then blank line, then more usage notes. Cover things like if a param is `None` then any triple matches; `subject` etc can be anything valid as input to `create_rdf_node`  typically an (NS, local) pair or string literal. Any or all may be `None` (if all are then every triple returned). Etc.